actions/src/cla_check/claCheck.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview File to check if PR Author has signed the CLA
+ */
+
+const core = require('@actions/core');
+const { context, GitHub } = require('@actions/github');
+const { google } = require('googleapis');
+const RANGE = 'Usernames';
+
+/**
+ * Create an OAuth2 client with the given credentials, and then execute the
+ * given claCheck function.
+ */
+const authorize = async function() {
+  const CREDENTIALS = JSON.parse(process.env.SHEETS_CRED);
+  const SHEETS_TOKEN = JSON.parse(process.env.SHEETS_TOKEN);
+  try {
+    // eslint-disable-next-line camelcase
+    const { client_secret, client_id, redirect_uris } = CREDENTIALS.installed;
+    const oAuth2Client = new google.auth.OAuth2(
+      client_id,
+      client_secret,
+      redirect_uris[0]
+    );
+    oAuth2Client.setCredentials(SHEETS_TOKEN);
+    return oAuth2Client;
+  } catch (err){
+    core.setFailed('Auth failure: ' + err);
+  }
+};
+
+const generateOutput = async (hasClaSigned) => {
+  const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
+  const LINK_RESULT = 'here'.link(
+    'https://github.com/oppia/oppia/wiki/' +
+    'Contributing-code-to-Oppia#setting-things-up'
+  );
+  const octokit = new GitHub(GITHUB_TOKEN);
+  const PR_NUMBER = context.payload.pull_request.number;
+  const PR_AUTHOR = context.payload.pull_request.user.login;
+
+  let comment = '';
+  if (!hasClaSigned) {
+    comment = ('Hi! @' +
+        PR_AUTHOR +
+        ' Welcome to Oppia! Please could you ' +
+        'follow the instructions ' + LINK_RESULT +
+        " to get started? You'll need to do " +
+        'this before we can accept your PR. Thanks!');
```

Comment:
Maybe specifically mention here that the CLA needs to be signed.

Reply:
How about this?
```
Hi @{{username}}, Welcome to Oppia. Could please follow the instruction here and sign the CLA Sheet to get started? You'll need to do this before we can accept your PR. Thanks!
```

Reply:
Sounds good.

Reply:
DONE